src/win/fs-event.c

```diff
@@ -455,36 +460,34 @@ void uv_process_fs_event_req(uv_loop_t* loop, uv_req_t* req,
             sizew = -1;
           }
 
-          if (filenamew) {
-            /* Convert the filename to utf8. */
+          /* Convert the filename to utf8. */
+          size = WideCharToMultiByte(CP_UTF8,
```

Comment:
ah, no can do. The buffer I have is not null terminated (yay Windows!) so we need to pass the size around, which `uv__convert_utf16_to_utf8` doesn't currently support.

I'll try to extend it in a follow-up PR.